src/components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,18 @@
+import './LandingPage.css'
+import landingPageBackground from '../../images/landing-page-background.jpg'
+
+const LandingPage = () => {
+  return (
+    <section className='landing-page-container'>
+      <div className='background-wash'>
+        <img src={ landingPageBackground } className='landing-page-img' alt='recycle icon with wooden background'></img>
```

Comment:
Let's break out lines over 80 characters onto separate lines for convention:
```suggestion
        <img
          src={ landingPageBackground }
          className='landing-page-img'
          alt='recycle icon with wooden background'
        >
         </img>
```